CRITICAL - 2025-08-20 15:49:32 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "vista_equipo.php"
[Method: GET, Route: buscar_equipo/1]
in SYSTEMPATH\Exceptions\FrameworkException.php on line 39.
 1 SYSTEMPATH\View\View.php(214): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('vista_equipo.php')
 2 SYSTEMPATH\Common.php(1167): CodeIgniter\View\View->render('vista_equipo', [], true)
 3 APPPATH\Controllers\EquipoController.php(41): view('vista_equipo', [...])
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\EquipoController->buscar('1')
 5 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EquipoController))
 6 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-08-20 15:51:40 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "vista_equipo.php"
[Method: GET, Route: buscar_equipo/501]
in SYSTEMPATH\Exceptions\FrameworkException.php on line 39.
 1 SYSTEMPATH\View\View.php(214): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('vista_equipo.php')
 2 SYSTEMPATH\Common.php(1167): CodeIgniter\View\View->render('vista_equipo', [], true)
 3 APPPATH\Controllers\EquipoController.php(41): view('vista_equipo', [...])
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\EquipoController->buscar('501')
 5 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EquipoController))
 6 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-08-20 15:53:25 --> ErrorException: Undefined variable $datos
[Method: GET, Route: buscar_equipo/501]
in APPPATH\Views\vista_equipos.php on line 85.
 1 APPPATH\Views\vista_equipos.php(85): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $datos', 'C:\\xampp\\htdocs\\fs2025\\codeigniter4-inventario\\app\\Views\\vista_equipos.php', 85)
 2 SYSTEMPATH\View\View.php(226): include('C:\\xampp\\htdocs\\fs2025\\codeigniter4-inventario\\app\\Views\\vista_equipos.php')
 3 SYSTEMPATH\View\View.php(229): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1167): CodeIgniter\View\View->render('vista_equipos', [], true)
 5 APPPATH\Controllers\EquipoController.php(41): view('vista_equipos', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\EquipoController->buscar('501')
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EquipoController))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-08-20 15:53:31 --> ErrorException: Undefined variable $datos
[Method: GET, Route: buscar_equipo/501]
in APPPATH\Views\vista_equipos.php on line 85.
 1 APPPATH\Views\vista_equipos.php(85): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $datos', 'C:\\xampp\\htdocs\\fs2025\\codeigniter4-inventario\\app\\Views\\vista_equipos.php', 85)
 2 SYSTEMPATH\View\View.php(226): include('C:\\xampp\\htdocs\\fs2025\\codeigniter4-inventario\\app\\Views\\vista_equipos.php')
 3 SYSTEMPATH\View\View.php(229): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1167): CodeIgniter\View\View->render('vista_equipos', [], true)
 5 APPPATH\Controllers\EquipoController.php(41): view('vista_equipos', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\EquipoController->buscar('501')
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EquipoController))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-08-20 15:54:47 --> mysqli_sql_exception: Table 'fs2025_inventario.equipo' doesn't exist in C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\BaseModel.php(715): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\fs2025\codeigniter4-inventario\app\Controllers\EquipoController.php(40): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\CodeIgniter.php(933): App\Controllers\EquipoController->buscar('501')
#8 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EquipoController))
#9 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\fs2025\codeigniter4-inventario\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}
CRITICAL - 2025-08-20 15:54:47 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'fs2025_inventario.equipo' doesn't exist
[Method: GET, Route: buscar_equipo/501]
in SYSTEMPATH\Database\BaseConnection.php on line 684.
 1 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `equipo`
WHERE `equipo_id` = :equipo_id:
 LIMIT 1', [...], false)
 2 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(715): CodeIgniter\Model->doFirst()
 4 APPPATH\Controllers\EquipoController.php(40): CodeIgniter\BaseModel->first()
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\EquipoController->buscar('501')
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EquipoController))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-08-20 15:54:47 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'fs2025_inventario.equipo' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `equipo`
WHERE `equipo_id` = \'501\'
 LIMIT 1')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `equipo`
WHERE `equipo_id` = \'501\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `equipo`
WHERE `equipo_id` = :equipo_id:
 LIMIT 1', [...], false)
 4 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(715): CodeIgniter\Model->doFirst()
 6 APPPATH\Controllers\EquipoController.php(40): CodeIgniter\BaseModel->first()
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\EquipoController->buscar('501')
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EquipoController))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-08-20 15:54:47 --> [Caused by] mysqli_sql_exception: Table 'fs2025_inventario.equipo' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT *
FROM `equipo`
WHERE `equipo_id` = \'501\'
 LIMIT 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `equipo`
WHERE `equipo_id` = \'501\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `equipo`
WHERE `equipo_id` = \'501\'
 LIMIT 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `equipo`
WHERE `equipo_id` = :equipo_id:
 LIMIT 1', [...], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(715): CodeIgniter\Model->doFirst()
 7 APPPATH\Controllers\EquipoController.php(40): CodeIgniter\BaseModel->first()
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\EquipoController->buscar('501')
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EquipoController))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-08-20 15:54:56 --> mysqli_sql_exception: Table 'fs2025_inventario.equipo' doesn't exist in C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\BaseModel.php(677): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\fs2025\codeigniter4-inventario\app\Controllers\EquipoController.php(10): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\CodeIgniter.php(933): App\Controllers\EquipoController->index()
#8 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EquipoController))
#9 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\fs2025\codeigniter4-inventario\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}
CRITICAL - 2025-08-20 15:54:56 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'fs2025_inventario.equipo' doesn't exist
[Method: GET, Route: equipos]
in SYSTEMPATH\Database\BaseConnection.php on line 684.
 1 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `equipo`', [], false)
 2 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(677): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\EquipoController.php(10): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\EquipoController->index()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EquipoController))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-08-20 15:54:56 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'fs2025_inventario.equipo' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `equipo`')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `equipo`')
 3 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `equipo`', [], false)
 4 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(677): CodeIgniter\Model->doFindAll(0, 0)
 6 APPPATH\Controllers\EquipoController.php(10): CodeIgniter\BaseModel->findAll()
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\EquipoController->index()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EquipoController))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-08-20 15:54:56 --> [Caused by] mysqli_sql_exception: Table 'fs2025_inventario.equipo' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT *
FROM `equipo`', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `equipo`')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `equipo`')
 4 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `equipo`', [], false)
 5 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(677): CodeIgniter\Model->doFindAll(0, 0)
 7 APPPATH\Controllers\EquipoController.php(10): CodeIgniter\BaseModel->findAll()
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\EquipoController->index()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EquipoController))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-08-20 15:55:02 --> mysqli_sql_exception: Table 'fs2025_inventario.equipo' doesn't exist in C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\BaseModel.php(677): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\fs2025\codeigniter4-inventario\app\Controllers\EquipoController.php(10): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\CodeIgniter.php(933): App\Controllers\EquipoController->index()
#8 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EquipoController))
#9 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\fs2025\codeigniter4-inventario\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}
CRITICAL - 2025-08-20 15:55:02 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'fs2025_inventario.equipo' doesn't exist
[Method: GET, Route: equipos]
in SYSTEMPATH\Database\BaseConnection.php on line 684.
 1 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `equipo`', [], false)
 2 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(677): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\EquipoController.php(10): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\EquipoController->index()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EquipoController))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-08-20 15:55:02 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'fs2025_inventario.equipo' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `equipo`')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `equipo`')
 3 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `equipo`', [], false)
 4 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(677): CodeIgniter\Model->doFindAll(0, 0)
 6 APPPATH\Controllers\EquipoController.php(10): CodeIgniter\BaseModel->findAll()
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\EquipoController->index()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EquipoController))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-08-20 15:55:02 --> [Caused by] mysqli_sql_exception: Table 'fs2025_inventario.equipo' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT *
FROM `equipo`', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `equipo`')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `equipo`')
 4 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `equipo`', [], false)
 5 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(677): CodeIgniter\Model->doFindAll(0, 0)
 7 APPPATH\Controllers\EquipoController.php(10): CodeIgniter\BaseModel->findAll()
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\EquipoController->index()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EquipoController))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-08-20 15:55:05 --> mysqli_sql_exception: Table 'fs2025_inventario.equipo' doesn't exist in C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\BaseModel.php(677): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\fs2025\codeigniter4-inventario\app\Controllers\EquipoController.php(10): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\CodeIgniter.php(933): App\Controllers\EquipoController->index()
#8 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EquipoController))
#9 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\fs2025\codeigniter4-inventario\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}
CRITICAL - 2025-08-20 15:55:05 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'fs2025_inventario.equipo' doesn't exist
[Method: GET, Route: equipos]
in SYSTEMPATH\Database\BaseConnection.php on line 684.
 1 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `equipo`', [], false)
 2 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(677): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\EquipoController.php(10): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\EquipoController->index()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EquipoController))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-08-20 15:55:05 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'fs2025_inventario.equipo' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `equipo`')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `equipo`')
 3 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `equipo`', [], false)
 4 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(677): CodeIgniter\Model->doFindAll(0, 0)
 6 APPPATH\Controllers\EquipoController.php(10): CodeIgniter\BaseModel->findAll()
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\EquipoController->index()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EquipoController))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-08-20 15:55:05 --> [Caused by] mysqli_sql_exception: Table 'fs2025_inventario.equipo' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT *
FROM `equipo`', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(729): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `equipo`')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `equipo`')
 4 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `equipo`', [], false)
 5 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(677): CodeIgniter\Model->doFindAll(0, 0)
 7 APPPATH\Controllers\EquipoController.php(10): CodeIgniter\BaseModel->findAll()
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\EquipoController->index()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EquipoController))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-08-20 15:55:35 --> ErrorException: Undefined variable $equipo
[Method: GET, Route: equipos]
in APPPATH\Views\vista_equipos.php on line 88.
 1 APPPATH\Views\vista_equipos.php(88): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $equipo', 'C:\\xampp\\htdocs\\fs2025\\codeigniter4-inventario\\app\\Views\\vista_equipos.php', 88)
 2 SYSTEMPATH\View\View.php(226): include('C:\\xampp\\htdocs\\fs2025\\codeigniter4-inventario\\app\\Views\\vista_equipos.php')
 3 SYSTEMPATH\View\View.php(229): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1167): CodeIgniter\View\View->render('vista_equipos', [], true)
 5 APPPATH\Controllers\EquipoController.php(11): view('vista_equipos', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\EquipoController->index()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EquipoController))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-08-20 15:56:13 --> ErrorException: Undefined variable $equipo
[Method: GET, Route: equipos]
in APPPATH\Views\vista_equipos.php on line 97.
 1 APPPATH\Views\vista_equipos.php(97): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $equipo', 'C:\\xampp\\htdocs\\fs2025\\codeigniter4-inventario\\app\\Views\\vista_equipos.php', 97)
 2 SYSTEMPATH\View\View.php(226): include('C:\\xampp\\htdocs\\fs2025\\codeigniter4-inventario\\app\\Views\\vista_equipos.php')
 3 SYSTEMPATH\View\View.php(229): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1167): CodeIgniter\View\View->render('vista_equipos', [], true)
 5 APPPATH\Controllers\EquipoController.php(11): view('vista_equipos', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\EquipoController->index()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EquipoController))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-08-20 15:56:37 --> ErrorException: Undefined variable $datos
[Method: GET, Route: buscar_equipo/501]
in APPPATH\Views\vista_equipos.php on line 85.
 1 APPPATH\Views\vista_equipos.php(85): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $datos', 'C:\\xampp\\htdocs\\fs2025\\codeigniter4-inventario\\app\\Views\\vista_equipos.php', 85)
 2 SYSTEMPATH\View\View.php(226): include('C:\\xampp\\htdocs\\fs2025\\codeigniter4-inventario\\app\\Views\\vista_equipos.php')
 3 SYSTEMPATH\View\View.php(229): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1167): CodeIgniter\View\View->render('vista_equipos', [], true)
 5 APPPATH\Controllers\EquipoController.php(41): view('vista_equipos', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\EquipoController->buscar('501')
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EquipoController))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-08-20 16:01:00 --> ErrorException: Undefined variable $datos
[Method: GET, Route: buscar_equipo/501]
in APPPATH\Views\vista_equipos.php on line 85.
 1 APPPATH\Views\vista_equipos.php(85): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $datos', 'C:\\xampp\\htdocs\\fs2025\\codeigniter4-inventario\\app\\Views\\vista_equipos.php', 85)
 2 SYSTEMPATH\View\View.php(226): include('C:\\xampp\\htdocs\\fs2025\\codeigniter4-inventario\\app\\Views\\vista_equipos.php')
 3 SYSTEMPATH\View\View.php(229): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1167): CodeIgniter\View\View->render('vista_equipos', [], true)
 5 APPPATH\Controllers\EquipoController.php(41): view('vista_equipos', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\EquipoController->buscar('501')
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EquipoController))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-08-20 16:01:01 --> ErrorException: Undefined variable $datos
[Method: GET, Route: buscar_equipo/501]
in APPPATH\Views\vista_equipos.php on line 85.
 1 APPPATH\Views\vista_equipos.php(85): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $datos', 'C:\\xampp\\htdocs\\fs2025\\codeigniter4-inventario\\app\\Views\\vista_equipos.php', 85)
 2 SYSTEMPATH\View\View.php(226): include('C:\\xampp\\htdocs\\fs2025\\codeigniter4-inventario\\app\\Views\\vista_equipos.php')
 3 SYSTEMPATH\View\View.php(229): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1167): CodeIgniter\View\View->render('vista_equipos', [], true)
 5 APPPATH\Controllers\EquipoController.php(41): view('vista_equipos', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\EquipoController->buscar('501')
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EquipoController))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-08-20 16:01:01 --> ErrorException: Undefined variable $datos
[Method: GET, Route: buscar_equipo/501]
in APPPATH\Views\vista_equipos.php on line 85.
 1 APPPATH\Views\vista_equipos.php(85): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $datos', 'C:\\xampp\\htdocs\\fs2025\\codeigniter4-inventario\\app\\Views\\vista_equipos.php', 85)
 2 SYSTEMPATH\View\View.php(226): include('C:\\xampp\\htdocs\\fs2025\\codeigniter4-inventario\\app\\Views\\vista_equipos.php')
 3 SYSTEMPATH\View\View.php(229): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1167): CodeIgniter\View\View->render('vista_equipos', [], true)
 5 APPPATH\Controllers\EquipoController.php(41): view('vista_equipos', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\EquipoController->buscar('501')
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EquipoController))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-08-20 16:01:01 --> ErrorException: Undefined variable $datos
[Method: GET, Route: buscar_equipo/501]
in APPPATH\Views\vista_equipos.php on line 85.
 1 APPPATH\Views\vista_equipos.php(85): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $datos', 'C:\\xampp\\htdocs\\fs2025\\codeigniter4-inventario\\app\\Views\\vista_equipos.php', 85)
 2 SYSTEMPATH\View\View.php(226): include('C:\\xampp\\htdocs\\fs2025\\codeigniter4-inventario\\app\\Views\\vista_equipos.php')
 3 SYSTEMPATH\View\View.php(229): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1167): CodeIgniter\View\View->render('vista_equipos', [], true)
 5 APPPATH\Controllers\EquipoController.php(41): view('vista_equipos', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\EquipoController->buscar('501')
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EquipoController))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-08-20 16:22:21 --> Error connecting to the database: mysqli_sql_exception: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión in C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Database\MySQLi\Connection.php:201
Stack trace:
#0 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Database\MySQLi\Connection.php(201): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'fs2025_inventar...', 3306, '', 0)
#1 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Database\BaseConnection.php(419): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `equipos...', Array, false)
#4 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\BaseModel.php(1025): CodeIgniter\Model->doUpdate(Array, Array)
#6 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Model.php(860): CodeIgniter\BaseModel->update(Array, Array)
#7 C:\xampp\htdocs\fs2025\codeigniter4-inventario\app\Controllers\EquipoController.php(57): CodeIgniter\Model->update('501', Array)
#8 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\CodeIgniter.php(933): App\Controllers\EquipoController->editar()
#9 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EquipoController))
#10 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\fs2025\codeigniter4-inventario\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión in C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Database\MySQLi\Connection.php:246
Stack trace:
#0 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Database\BaseConnection.php(419): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `equipos...', Array, false)
#3 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
#4 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\BaseModel.php(1025): CodeIgniter\Model->doUpdate(Array, Array)
#5 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Model.php(860): CodeIgniter\BaseModel->update(Array, Array)
#6 C:\xampp\htdocs\fs2025\codeigniter4-inventario\app\Controllers\EquipoController.php(57): CodeIgniter\Model->update('501', Array)
#7 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\CodeIgniter.php(933): App\Controllers\EquipoController->editar()
#8 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EquipoController))
#9 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\fs2025\codeigniter4-inventario\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}
CRITICAL - 2025-08-20 16:22:21 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión
[Method: POST, Route: editar_equipo]
in SYSTEMPATH\Database\BaseConnection.php on line 465.
 1 SYSTEMPATH\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `equipos` SET `equipo_id` = :equipo_id:, `no_serie` = :no_serie:, `marca_id` = :marca_id:, `descripcion` = :descripcion:, `fecha_compra` = :fecha_compra:, `precio` = :precio:, `tipo_equipo` = :tipo_equipo:, `empleado_id` = :empleado_id:
WHERE `equipos`.`equipo_id` IN :equipos.equipo_id:', [...], false)
 3 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 4 SYSTEMPATH\BaseModel.php(1025): CodeIgniter\Model->doUpdate([...], [...])
 5 SYSTEMPATH\Model.php(860): CodeIgniter\BaseModel->update([...], [...])
 6 APPPATH\Controllers\EquipoController.php(57): CodeIgniter\Model->update('501', [...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\EquipoController->editar()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EquipoController))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-08-20 16:22:35 --> Error connecting to the database: mysqli_sql_exception: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión in C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Database\MySQLi\Connection.php:201
Stack trace:
#0 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Database\MySQLi\Connection.php(201): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'fs2025_inventar...', 3306, '', 0)
#1 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Database\BaseConnection.php(419): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `equipos...', Array, false)
#4 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\BaseModel.php(1025): CodeIgniter\Model->doUpdate(Array, Array)
#6 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Model.php(860): CodeIgniter\BaseModel->update(Array, Array)
#7 C:\xampp\htdocs\fs2025\codeigniter4-inventario\app\Controllers\EquipoController.php(57): CodeIgniter\Model->update('501', Array)
#8 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\CodeIgniter.php(933): App\Controllers\EquipoController->editar()
#9 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EquipoController))
#10 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\fs2025\codeigniter4-inventario\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión in C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Database\MySQLi\Connection.php:246
Stack trace:
#0 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Database\BaseConnection.php(419): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `equipos...', Array, false)
#3 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
#4 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\BaseModel.php(1025): CodeIgniter\Model->doUpdate(Array, Array)
#5 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Model.php(860): CodeIgniter\BaseModel->update(Array, Array)
#6 C:\xampp\htdocs\fs2025\codeigniter4-inventario\app\Controllers\EquipoController.php(57): CodeIgniter\Model->update('501', Array)
#7 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\CodeIgniter.php(933): App\Controllers\EquipoController->editar()
#8 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EquipoController))
#9 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\fs2025\codeigniter4-inventario\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}
CRITICAL - 2025-08-20 16:22:35 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión
[Method: POST, Route: editar_equipo]
in SYSTEMPATH\Database\BaseConnection.php on line 465.
 1 SYSTEMPATH\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `equipos` SET `equipo_id` = :equipo_id:, `no_serie` = :no_serie:, `marca_id` = :marca_id:, `descripcion` = :descripcion:, `fecha_compra` = :fecha_compra:, `precio` = :precio:, `tipo_equipo` = :tipo_equipo:, `empleado_id` = :empleado_id:
WHERE `equipos`.`equipo_id` IN :equipos.equipo_id:', [...], false)
 3 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 4 SYSTEMPATH\BaseModel.php(1025): CodeIgniter\Model->doUpdate([...], [...])
 5 SYSTEMPATH\Model.php(860): CodeIgniter\BaseModel->update([...], [...])
 6 APPPATH\Controllers\EquipoController.php(57): CodeIgniter\Model->update('501', [...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\EquipoController->editar()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EquipoController))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-08-20 16:22:53 --> Error connecting to the database: mysqli_sql_exception: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión in C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Database\MySQLi\Connection.php:201
Stack trace:
#0 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Database\MySQLi\Connection.php(201): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'fs2025_inventar...', 3306, '', 0)
#1 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Database\BaseConnection.php(419): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `equipos...', Array, false)
#4 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\BaseModel.php(1025): CodeIgniter\Model->doUpdate(Array, Array)
#6 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Model.php(860): CodeIgniter\BaseModel->update(Array, Array)
#7 C:\xampp\htdocs\fs2025\codeigniter4-inventario\app\Controllers\EquipoController.php(57): CodeIgniter\Model->update('501', Array)
#8 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\CodeIgniter.php(933): App\Controllers\EquipoController->editar()
#9 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EquipoController))
#10 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\fs2025\codeigniter4-inventario\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión in C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Database\MySQLi\Connection.php:246
Stack trace:
#0 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Database\BaseConnection.php(419): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `equipos...', Array, false)
#3 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
#4 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\BaseModel.php(1025): CodeIgniter\Model->doUpdate(Array, Array)
#5 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Model.php(860): CodeIgniter\BaseModel->update(Array, Array)
#6 C:\xampp\htdocs\fs2025\codeigniter4-inventario\app\Controllers\EquipoController.php(57): CodeIgniter\Model->update('501', Array)
#7 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\CodeIgniter.php(933): App\Controllers\EquipoController->editar()
#8 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EquipoController))
#9 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\fs2025\codeigniter4-inventario\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}
CRITICAL - 2025-08-20 16:22:53 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión
[Method: POST, Route: editar_equipo]
in SYSTEMPATH\Database\BaseConnection.php on line 465.
 1 SYSTEMPATH\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `equipos` SET `equipo_id` = :equipo_id:, `no_serie` = :no_serie:, `marca_id` = :marca_id:, `descripcion` = :descripcion:, `fecha_compra` = :fecha_compra:, `precio` = :precio:, `tipo_equipo` = :tipo_equipo:, `empleado_id` = :empleado_id:
WHERE `equipos`.`equipo_id` IN :equipos.equipo_id:', [...], false)
 3 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 4 SYSTEMPATH\BaseModel.php(1025): CodeIgniter\Model->doUpdate([...], [...])
 5 SYSTEMPATH\Model.php(860): CodeIgniter\BaseModel->update([...], [...])
 6 APPPATH\Controllers\EquipoController.php(57): CodeIgniter\Model->update('501', [...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\EquipoController->editar()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EquipoController))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-08-20 16:23:02 --> Error connecting to the database: mysqli_sql_exception: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión in C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Database\MySQLi\Connection.php:201
Stack trace:
#0 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Database\MySQLi\Connection.php(201): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'fs2025_inventar...', 3306, '', 0)
#1 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Database\BaseConnection.php(419): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `equipos...', Array, false)
#4 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
#5 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\BaseModel.php(1025): CodeIgniter\Model->doUpdate(Array, Array)
#6 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Model.php(860): CodeIgniter\BaseModel->update(Array, Array)
#7 C:\xampp\htdocs\fs2025\codeigniter4-inventario\app\Controllers\EquipoController.php(57): CodeIgniter\Model->update('501', Array)
#8 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\CodeIgniter.php(933): App\Controllers\EquipoController->editar()
#9 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EquipoController))
#10 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\fs2025\codeigniter4-inventario\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión in C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Database\MySQLi\Connection.php:246
Stack trace:
#0 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Database\BaseConnection.php(419): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `equipos...', Array, false)
#3 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
#4 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\BaseModel.php(1025): CodeIgniter\Model->doUpdate(Array, Array)
#5 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Model.php(860): CodeIgniter\BaseModel->update(Array, Array)
#6 C:\xampp\htdocs\fs2025\codeigniter4-inventario\app\Controllers\EquipoController.php(57): CodeIgniter\Model->update('501', Array)
#7 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\CodeIgniter.php(933): App\Controllers\EquipoController->editar()
#8 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EquipoController))
#9 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\fs2025\codeigniter4-inventario\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\fs2025\codeigniter4-inventario\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}
CRITICAL - 2025-08-20 16:23:02 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión
[Method: POST, Route: editar_equipo]
in SYSTEMPATH\Database\BaseConnection.php on line 465.
 1 SYSTEMPATH\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2521): CodeIgniter\Database\BaseConnection->query('UPDATE `equipos` SET `equipo_id` = :equipo_id:, `no_serie` = :no_serie:, `marca_id` = :marca_id:, `descripcion` = :descripcion:, `fecha_compra` = :fecha_compra:, `precio` = :precio:, `tipo_equipo` = :tipo_equipo:, `empleado_id` = :empleado_id:
WHERE `equipos`.`equipo_id` IN :equipos.equipo_id:', [...], false)
 3 SYSTEMPATH\Model.php(461): CodeIgniter\Database\BaseBuilder->update()
 4 SYSTEMPATH\BaseModel.php(1025): CodeIgniter\Model->doUpdate([...], [...])
 5 SYSTEMPATH\Model.php(860): CodeIgniter\BaseModel->update([...], [...])
 6 APPPATH\Controllers\EquipoController.php(57): CodeIgniter\Model->update('501', [...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\EquipoController->editar()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EquipoController))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-08-20 16:58:56 --> ErrorException: Undefined variable $request
[Method: POST, Route: agregarMarca]
in APPPATH\Controllers\MarcaController.php on line 18.
 1 APPPATH\Controllers\MarcaController.php(18): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $request', 'C:\\xampp\\htdocs\\fs2025\\codeigniter4-inventario\\app\\Controllers\\MarcaController.php', 18)
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\MarcaController->agregarMarca()
 3 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\MarcaController))
 4 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
